gravitee-apim-cypress/cypress/commands/management/api-management-commands.ts

```diff
@@ -43,3 +45,35 @@ export function deleteApi(auth: BasicAuthentication, apiId: string) {
     failOnStatusCode: false,
   });
 }
+
+export function importCreateApi(auth: BasicAuthentication, body: ApiImport) {
+  return cy.request({
+    method: 'POST',
+    url: `${Cypress.config().baseUrl}${Cypress.env('managementApi')}/apis/import`,
+    body,
+    auth,
+    failOnStatusCode: false,
+  });
+}
+
+export function importUpdateApi(auth: BasicAuthentication, apiId: string, body: ApiImport) {
+  return cy.request({
+    method: 'PUT',
+    url: `${Cypress.config().baseUrl}${Cypress.env('managementApi')}/apis/${apiId}/import`,
+    body,
+    auth,
+    failOnStatusCode: false,
+  });
+}
+
+export function getPages(auth: BasicAuthentication, apiId: string) {
+  return cy.request({
+    method: 'GET',
+    url: `${Cypress.config().baseUrl}${Cypress.env('managementApi')}/apis/${apiId}/pages`,
+    auth,
+    failOnStatusCode: false,
+    qs: {
+      root: true,
+    },
+  });
+}
```

Comment:
I would prefer a file api-pages-management-commands.ts to avoid mixing concepts.
wdyt ?

Reply:
Yes, has been splitted. Thanks.